tests/models/whisper/test_modeling_tf_whisper.py

```diff
@@ -717,41 +890,10 @@ def test_small_en_logits_librispeech(self):
         # fmt: on
         self.assertTrue(np.allclose(logits[0, 0, :30], EXPECTED_LOGITS, atol=1e-4))
 
+    @run_test_in_subprocess(target_func=_test_large_logits_librispeech, timeout=os.environ.get("PYTEST_TIMEOUT", 600))
     @slow
     def test_large_logits_librispeech(self):
-        set_seed(0)
-
-        model = TFWhisperModel.from_pretrained("openai/whisper-large")
-
-        input_speech = self._load_datasamples(1)
-
-        processor = WhisperProcessor.from_pretrained("openai/whisper-large")
-        processed_inputs = processor(audio=input_speech, text="This part of the speech", return_tensors="tf")
-        input_features = processed_inputs.input_features
-        labels = processed_inputs.labels
-
-        logits = model(
-            input_features,
-            decoder_input_ids=labels,
-            output_hidden_states=False,
-            output_attentions=False,
-            use_cache=False,
-        )
-
-        logits = logits.last_hidden_state @ tf.transpose(model.model.decoder.embed_tokens.weights[0])
-
-        # fmt: off
-        EXPECTED_LOGITS = tf.convert_to_tensor(
-            [
-                2.1382, 0.9381, 4.4671, 3.5589, 2.4022, 3.8576, -0.6521, 2.5472,
-                1.8301, 1.9957, 2.3432, 1.4678, 0.5459, 2.2597, 1.5179, 2.5357,
-                1.1624, 0.6194, 1.0757, 1.8259, 2.4076, 1.6601, 2.3503, 1.3376,
-                1.9891, 1.8635, 3.8931, 5.3699, 4.4772, 3.9184
-            ]
-        )
-        # fmt: on
-
-        self.assertTrue(np.allclose(logits[0, 0, :30], EXPECTED_LOGITS, atol=1e-4))
+        pass
```

Comment:
This doesn't mean the test does nothing. The actual testing logic is in `_test_large_logits_librispeech` and being launched in the decorator `run_test_in_subprocess`.

This is somehow confusing, but it works well. I can add comment to avoid confusion if we decide to go with this approach.

@sgugger Could you take a look and let me which approach you think better.

Reply:
The whole goal of the decorator is to not specuf the `target_func` and apply it to the `_test_large_logits_librispeech` function. ;-)